examples/commandTest.lean

```diff
@@ -2,36 +2,34 @@ import Qq
 
 open Qq
 
+macro "trace_state" : doElem => `(doElem| pure (by trace_state; exact ()))
+
 -- using the goal type (interpret 2, 3 as Int)
 /--
-info: x
----
-info: Prop
+trace: x : Q(Prop)
+inst✝ : Q(Decidable «$x»)
+⊢ PUnit
```

Comment:
This is supposed to be showing `Q(Int)` or `TermElab Q(Int)`...

Reply:
well no, because it is a single line of the do block which doesn't return anything

Reply:
Ah, then I guess `trace_state` is wrong.